imports/server/discord.ts

```diff
@@ -184,6 +184,7 @@ class DiscordBot {
 }
 
 const handleOauthRequest = (query: any) => {
+  // eslint-disable-next-line jolly-roger/no-sync-mongo-methods
```

Comment:
Also implemented upstream but blocked on the Meteor 2.9 update: https://github.com/meteor/meteor/pull/12156/files#diff-6cc58ccee99c3992649e7b63e5df7590bdaae86d9dca037b2e573227b2249552R10